enhancements/oc/must-gather-collection-mode.md

```diff
@@ -0,0 +1,211 @@
+---
+title: must-gather-collection-mode
+authors:
+  - "@ardaguclu"
+reviewers:
+  - "@ingvagabund"
+approvers: 
+  - "@deads2k"
+api-approvers: 
+  - "@deads2k"
+creation-date: 2024-11-22
+last-updated: 2024-11-22
+tracking-link: 
+  - https://issues.redhat.com/browse/OCPBUGS-37344
+see-also:
+  - "/enhancements/oc/must-gather.md"
+  - "/enhancements/oc/inspect.md"
+---
+
+# must-gather: Collection Mode
+
+## Summary
+
+This proposal introduces a new flag (i.e. `--collection-mode`) in must-gather targeting to large clusters by skipping some
+logs to take less time and storage size. In addition to that, this proposal introduces a new `--node-selector` flag
```

Comment:
@deads2k actually, I'd see more escalable to be able to provide arbitrary parameters to be passed to any `oc adm inspect` command. I know that adhering to this would depend on each and every must-gather image (and I'd say we can live with non-default images adopting this as something optional and/or at different paces). However, having to coordinate each change between `oc adm inspect`, `oc adm must-gather` and the different must-gather images looks like a much more complex process to me.